src/components/VideoCard/VideoCard.css

```diff
@@ -0,0 +1,48 @@
+.video-card {
+  margin: 0;
+  background-color: inherit;
+  cursor: pointer;
+}
+
+.video-card:hover {
+  box-shadow: var(--box-shadow);
+}
+
+.video-card,
+.video-card .card-img {
+  border-radius: 0;
+}
+
+.video-card .card-content {
+  max-width: 100%;
+  display: flex;
+  align-items: flex-start;
+  gap: 0.5rem;
+  padding: 0.5rem 0;
+}
+
+.video-card .card-content .card-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex-grow: 1;
+}
+
+.video-card .card-heading {
+  padding: 0;
+  font-size: 0.9rem;
+  font-weight: bold;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
```

Comment:
The use of webkit property often generates cross-platform issues, maybe also give a check if it works fine across different browsers

Reply:
Yes, I checked. It works well in other browsers as well. Thank you.